### PR TITLE
Srp 469

### PR DESCRIPTION
**Pull request by hcvdwerf**

### CODE REVIEW

File: src/public/web.config [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+<configuration>
+  <system.webServer>
+    <rewrite>
+      <rules>
+        <rule name="Redirect subdomains to healthdata.nl" stopProcessing="true">
+          <match url=".*" />
+          <conditions>
+            <add input="{HTTP_HOST}" pattern="^(catalog|catalogue|catalogus)\.healthdata\.nl$" />
+          </conditions>
+          <action type="Redirect" url="http://healthdata.nl/{R:0}" redirectType="Permanent" />∏
```
Comment (Markus92) [inline src/public/web.config]: Should be https not http

File: src/public/web.config.license [hunk 2]
```diff
@@ -0,0 +1,3 @@
+SPDX-FileCopyrightText: 2024 Unsplash
```
Comment (Markus92) [inline src/public/web.config.license]: File is not copyright Unsplash (it's not a stock photo) but Health-RI? License should be updated to Apache 2.0

**Review by Markus92 — CHANGES_REQUESTED** [review]
Two small things, looks good otherwise!

File: src/public/web.config (hunk 1 above)
Comment (Markus92) [reply src/public/web.config]: And you might want to watch "www." too?

**Review by Markus92 — APPROVED** [review]
LGTM

File: src/public/web.config (hunk 1 above)
Comment (hcvdwerf) [reply src/public/web.config]: yes added now

**Pull request merged**